pkg/cmd/roachtest/tests/restore_test.go

```diff
@@ -53,24 +53,19 @@ func TestRestoreRegisteredNames(t *testing.T) {
 	expectedRestoreTests := []string{
 		"restore/nodeShutdown/coordinator",
 		"restore/nodeShutdown/worker",
-		"restore/pause/tpcc-5k-350GiB/aws/inc-count=12/nodes=4/cpus=8",
-		"restore/tpcc-10-700MiB/gce/inc-count=3/nodes=4/cpus=8",
-		"restore/tpcc-300k-20TiB/aws/inc-count=12/nodes=15/cpus=16",
-		"restore/tpcc-300k-20TiB/aws/inc-count=400/nodes=15/cpus=16",
-		"restore/tpcc-300k-20TiB/gce/inc-count=400/nodes=15/cpus=16",
-		"restore/tpcc-30k-2TiB/aws/inc-count=12/nodes=4/cpus=16",
-		"restore/tpcc-30k-2TiB/aws/inc-count=12/nodes=4/cpus=8",
-		"restore/tpcc-30k-2TiB/aws/inc-count=12/nodes=8/cpus=8",
-		"restore/tpcc-30k-2TiB/aws/inc-count=12/nodes=9/cpus=8/zones=us-east-2b,us-west-2b,eu-west-1b",
-		"restore/tpcc-30k-2TiB/aws/inc-count=48/nodes=4/cpus=8",
-		"restore/tpcc-30k-2TiB/gce/inc-count=0/nodes=10/cpus=8",
-		"restore/tpcc-30k-2TiB/gce/inc-count=12/nodes=4/cpus=8",
-		"restore/tpcc-30k-2TiB/gce/inc-count=12/nodes=4/cpus=8/lowmem",
-		"restore/tpcc-5k-350GiB/gce/inc-count=0/nodes=4/cpus=8",
-		"restore/tpcc-5k-350GiB/gce/inc-count=48/nodes=4/cpus=8",
+		"restore/pause/tpcc-5k/gce/nodes=4/cpus=8",
+		"restore/tpcc-10/gce/nodes=4/cpus=8",
```

Comment:
So the name got simplified and doesn't include the size of the fixture (you'd have to look at our fixtures in `backup_fixture.go` to determine the size). Do you think we should still keep the size of the fixture here?

Reply:
could `TpccFixture.Name()` simply be `fmt.Sprintf(tpcc-%d-%s,ImportWarehouses,RestoredSizeEstimate)`

Reply:
Yeah it can be, was just wondering if we should just use `Kind`, or add another `Name` method to the interface. If the restore size wasn't something that would add much value to the roachtest name, I was thinking we could just skip it.

Reply:
oh i see. yeah i think it is fine as is. don't worry about the roachtest names.